### PR TITLE
compass.sql file ready for review

### PR DESCRIPTION
**Pull request by FatimaAbdimalik**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
Provide a short description of the change you are proposing and the reasons for it.

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: comapss.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  email VARCHAR(120) NOT NULL,
+  phone_number INT,
+  password VARCHAR(30) NOT NULL,
+  city VARCHAR(30) NOT NULL,
+  cohort VARCHAR(30) NOT NULL,
+  biography VARCHAR(500),
+  picture OID
+);
+
+CREATE TABLE MENTORS (
```
Comment (Dedekind561) [inline comapss.sql:19]:
```suggestion
CREATE TABLE mentors (
```

**Review by Dedekind561 — CHANGES_REQUESTED** [review]

File: comapss.sql [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ INSERT INTO students (name, surname, email, password, city, cohort,biography) VALUES ('Fatima', 'Abdimalik','f.abdimalik@gmail.com', '123','Manchester','north-west-class3', 'A tech enthusiast');
+ INSERT INTO students (name, surname, email, password, city, cohort,biography) VALUES ('Sulaiman', 'Solo','sulaimansolo73505@gmail.com', '123','Manchester','north-west-class3', 'software trainee');
+
+
+
+
+INSERT INTO mentors (name, email, password) VALUES ('Ahmad','ahmadshagel@gmail.com', 'efgh');
+
+
+INSERT INTO FEEDBACKTABLE (mentor_id, student_id, sent_date, title, body, response) VALUES(1,4,'2020-10-13','HTML feedback','Well done, good effort', 'Thank you mentor');
```
Comment (msachi) [inline comapss.sql:48]: This should now be `feedbacktable`

File: comapss.sql [hunk 3]
```diff
@@ -0,0 +1,48 @@
+drop table if exists students;
+drop table if exists mentors;
+drop table if exists feedbacktable;
```
Comment (msachi) [inline comapss.sql:3]: For consistency you can make all commands upper case: `DROP TABLE IF EXISTS`

File: comapss.sql [hunk 4]
```diff
@@ -0,0 +1,48 @@
+drop table if exists students;
```
Comment (msachi) [inline comapss.sql:1]: The name of the sql file is misspelled

**Pull request merged**